src/Tools/dotnet-dsrouter/ADBTcpRouterFactory.cs

```diff
@@ -148,6 +128,38 @@ void sdklogger(TraceLevel level, string message)
 
             return processStartedResult && expectedOutputResult && expectedExitCodeResult;
         }
+
+        private static string AndroidSdkPath;
```

Comment:
I'm open if someone has a better idea than static field, thanks.

Reply:
Its fine caching it, this is called when we run different adb commands in dsrouter, it also does a periodic check to make sure ports are still forwarded/reversed if it owns that, so expect this function to be called several times during startup + periodically while dsrouter is running.

Reply:
Yes, I noticed it continued logging those messages every second or so.